distribution/openhabhome/configurations/openhab_default.cfg

```diff
@@ -234,19 +234,34 @@ chart:provider=default
 #####                   I/O component configurations                              #####
 #######################################################################################
 
-########################## Google Calendar configuration ##############################
+####################### Google Calendar (GCal) configuration ##########################
 #
-# the username and password for Google Calendar Account
-#gcal:username=
-#gcal:password=
+# Before using GCal, you need to have a Google API
+# Console project. The Wiki describes in detail the steps necessary to set 
+# up your Google API Console project, as well as how to obtain the credentials 
+# necessary to complete the information in this file. Once the project is created, 
+# and you've completed the steps described in the Wiki, you need to copy 
+# the "Client ID" and "Client secret" from the Credentials page on 
+# console.developers.google.com
+#
+# Copied from the "Client ID" field on the Credentials page (required)
+#gcal:client_id=
+
+# Copied from the "Client secret" field on the Credentials page (required)
+#gcal:client_secret=
 
-# the url of the calendar feed
-#gcal:url=
+# This is the name you gave to your Google Calendar.  GCal will download calendar 
+# events from this calendar (required)
+#gcal:calendar_name=
```

Comment:
I've confirmed the following:
- both `gcal `and `gcal-persistence` require a calendar name in the cfg.  If not supplied, it doesn't default to anything.  It does log errors in the log file, however.
- using the keyword `primary `does use your default Google calendar.  My default calendar is now full of presence simulation events.  :-(

Therefore, I'm not sure it's a good idea to default to `primary `if no calendar_name is specified (at least for `gcal-presence`), unless one wants to discover a bunch of presence simulation events in their default Google calendar.

I would propose that I update the cfg comments (gcal, gcal-persistence, and openhab_default) to reflect the usage of the keyword `primary`, but leave `calendar_name` required.

Also, in order to get gcal-persistence to run in OH2, I needed to remove these imported packages from `MANIFEST.MF`.  Should I include an updated `MANIFEST.MF` in this PR?
```
ch.qos.logback.classic,
ch.qos.logback.classic.encoder,
ch.qos.logback.classic.spi,
ch.qos.logback.core,
ch.qos.logback.core.encoder,
ch.qos.logback.core.pattern,
ch.qos.logback.core.spi,
```